pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -351,14 +428,29 @@ async def _map_message(self, messages: list[ModelMessage]) -> tuple[str, list[Be
                                     thinking=response_part.content, signature=response_part.signature, type='thinking'
                                 )
                             )
-                    else:
-                        tool_use_block_param = BetaToolUseBlockParam(
+                    elif isinstance(response_part, ServerToolCallPart):
```

Comment:
I think we only want to include this if `response_part.model_name == 'anthropic'`, so we don't send builtin tool calls from one model/provider to another. (Note that I suggested renaming this to `provider_name` elsewhere)